0.6/conf/janusgraph-berkeleyje-es-server.properties

```diff
@@ -0,0 +1,30 @@
+#
+# NOTE: THIS FILE IS GENERATED VIA "update.sh"
+# DO NOT EDIT IT DIRECTLY; CHANGES WILL BE OVERWRITTEN.
+#
+# Copyright 2021 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# JanusGraph configuration sample: BerkeleyDB JE and embedded Elasticsearch
+#
+# This file opens a BDB JE instance in the directory
+# db/berkeley.  It also starts a local Elasticsearch
+# service inside the same JVM running JanusGraph, persisted at
+# db/es.
+
+gremlin.graph=org.janusgraph.core.JanusGraphFactory
+storage.backend=berkeleyje
+storage.directory=/var/lib/janusgraph/data
+index.search.backend=elasticsearch
+index.search.hostname=elasticsearch
```

Comment:
I didn't check but is it the right hostname? I see we use 'elasticsearch' hostname in some configurations and '127.0.0.1' in other configurations. Is it right?

Reply:
I changed it to elasticsearch in the normal case, es should be reachable under localhost.